### PR TITLE
VAGOV-5991: Adding vamc banner display logic

### PR DESCRIPTION
**Pull request by ethanteague**
## Description
Adds region specific banners to top level and all subordinate pages. Alert vamc targetting, dismissability, add cta's determined Drupal node settings (see screenshot).

## Testing done
Visual

## Screenshots
Screenshot from 2019-11-14 14-41-44

## Acceptance criteria
- [x] @kevwalsh review

### CODE REVIEW

File: src/site/components/banner_alerts.drupal.liquid [hunk 1]
```diff
@@ -0,0 +1,70 @@
+{% for entity in alert.entities %}
+
+{% assign alertType = entity.fieldAlertType %}
+
+{% if alertType = "information" %}
+{% assign alertType = "info" %}
```
Comment (swirtSJW) [inline src/site/components/banner_alerts.drupal.liquid]: CS: SHouldn't these be indented inside the control structure?

File: src/site/components/banner_alerts.drupal.liquid [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+{% assign outputStatus = false %}
+{% assign emailUpdates = "https://www.va.gov/" %}
+{% assign statusUrl = "https://www.va.gov/" %}
+
+{% for vamc in entity.fieldBannerAlertVamcs %}
+
+{% if region == vamc.entity.fieldOffice.entity.entityUrl.path %}
+{% assign outputStatus = true %}
+{% assign emailUpdates = vamc.entity.fieldOffice.entity.fieldLinkFacilityEmergList.uri %}
+{% assign statusUrl = vamc.entity.entityUrl.path %}
```
Comment (swirtSJW) [inline src/site/components/banner_alerts.drupal.liquid]: CS indentation?

**Review by swirtSJW — APPROVED** [review]
Looks good but I am not sure how strict the CS for control structures should be on liquid templates.  It would  greatly improve readability.

File: src/site/stages/build/drupal/graphql/GetAllPages.graphql.js [hunk 3]
```diff
@@ -9,7 +9,7 @@ const vamcOperatingStatusAndAlerts = require('./vamcOperatingStatusAndAlerts.gra
 const fragments = require('./fragments.graphql');
 const newsStoryPage = require('./newStoryPage.graphql');
 const sidebarQuery = require('./navigation-fragments/sidebar.nav.graphql');
-const alertsQuery = require('./alerts.graphql');
```
Comment (rianfowler) [inline src/site/stages/build/drupal/graphql/GetAllPages.graphql.js]: Should we delete this `alerts.graphql` as part of this PR? looks like both the references to it are removed with this work.

File: src/site/components/banner_alerts.drupal.liquid (hunk 1 above)
Comment (ethanteague) [reply src/site/components/banner_alerts.drupal.liquid]: I'm not sure what the standard is, @swirtSJW - seems like we are all over the place in our liquid files. Fwiw, my VsCode liquid formatter thinks that this is the way it's supposed to look (e.g., indenting code within conditionals manually is pulled back flush after autofomatting).

File: src/site/stages/build/drupal/graphql/GetAllPages.graphql.js (hunk 3 above)
Comment (ethanteague) [reply src/site/stages/build/drupal/graphql/GetAllPages.graphql.js]: :+1: @rianfowler

File: src/site/components/banner_alerts.drupal.liquid [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+{% assign outputStatus = false %}
+{% endif %}
+{% assign emailUpdates = vamc.entity.fieldOffice.entity.fieldLinkFacilityEmergList.uri %}
+{% assign statusUrl = vamc.entity.entityUrl.path %}
+{% endif %}
+{% endfor %}
+
+{% if outputStatus == true %}
+<div aria-labelledby="usa-alert-heading-{{ entity.entityId }}"
+  class="usa-alert-full-width dismissable-option-header usa-alert-full-width-{{ entity.fieldAlertType }}"
```
Comment (cvalarida) [inline src/site/components/banner_alerts.drupal.liquid]: Looks like we're using `alertType` elsewhere. Should this follow suit?

File: src/site/components/banner_alerts.drupal.liquid (hunk 1 above)
Comment (cvalarida) [reply src/site/components/banner_alerts.drupal.liquid]: I think it makes sense to keep it indented like it was "normal" code. Ideally, we'd lint this, but we haven't gotten there yet. Without the indentation, it makes the following pretty tough to read.

File: src/site/components/banner_alerts.drupal.liquid [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+    <div class="usa-alert-body">
+      <h3 class="usa-alert-heading">
+        {{ entity.title }}
+      </h3>
+      <div class="additional-info-content usa-alert-text">
+        {{ entity.fieldBody.processed }}
+
+        {% if entity.fieldAlertOperatingStatusCta == true %}
+        <p>
+          <a href="{{statusUrl}}">Get
```
Comment (cvalarida) [inline src/site/components/banner_alerts.drupal.liquid]: Does this work, or does it need spaces around the variable name? (`{{ statusUrl }}`)

File: src/site/components/banner_alerts.drupal.liquid (hunk 4 above)
Comment (ethanteague) [reply src/site/components/banner_alerts.drupal.liquid]: Updated @cvalarida :+1:

File: src/site/components/banner_alerts.drupal.liquid (hunk 5 above)
Comment (ethanteague) [reply src/site/components/banner_alerts.drupal.liquid]: It does, but Updated @cvalarida :+1:

File: src/site/components/banner_alerts.drupal.liquid (hunk 2 above)
Comment (ethanteague) [reply src/site/components/banner_alerts.drupal.liquid]: Updated @swirtSJW :+1:

File: src/site/components/banner_alerts.drupal.liquid (hunk 1 above)
Comment (ethanteague) [reply src/site/components/banner_alerts.drupal.liquid]: Updated @swirtSJW @cvalarida :+1:

**Review by kevwalsh — APPROVED** [review]
LGTM

**Review by cvalarida — APPROVED** [review]
Looks good, from what I can tell. Sorry for the wait! :grimacing:

**Pull request merged**